code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -2085,12 +2085,18 @@
 		name = "Spiced Rum"
 		id = "spicedrum"
 		result = "spicedrum"
-		required_reagents = list("rum" = 1, "capsaicin" = 1)
+		required_reagents = list("rum" = 1, "cinnamon" = 1)
 		result_amount = 2
-		mix_phrase = "You feel like you might have misunderstood the recipe."
```

Comment:
Eh, this is kind of a funny joke. Idk if we want a weird fake version idk

Reply:
The fake version doesn't actually act any different, it just has the same old mix phrase it used to have before. This mostly just lets you make spiced rum with cinnamon and capsaicin. I made it a separate chem for fun future interactions, but if preferred they can both just result in the same chem.